mycore-viewer/src/main/typescript/modules/base/MyCoReViewerSettings.ts

```diff
@@ -48,7 +48,12 @@ namespace mycore.viewer {
 
             settings.filePath = encodeURI(settings.filePath);
 
-            if(settings.webApplicationBaseURL.lastIndexOf("/")==settings.webApplicationBaseURL.length-1){
+            if (settings.filePath.indexOf('_derivate_') > 0) {
+                settings.filePath = settings.filePath.replace('/', '%2F');
+            }
+
+            if(settings.webApplicationBaseURL !== "undefined"
+                && settings.webApplicationBaseURL.lastIndexOf("/") === settings.webApplicationBaseURL.length - 1){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [" should be '](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4613896)